### DISCUSSION TITLE
Separate Translations in Uno Library Projects

### DISCUSSION THREAD

**Discussion by DavidM29** [unoplatform/uno#18445] [Q&A] [answered]
Is it possible to have `.resw` files in Uno Library Projects?

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the question. Yes, it's possible to do that. We have documentation for [localization here](https://platform.uno/docs/articles/features/working-with-strings.html), as well as [microsoft's one on the topic](https://learn.microsoft.com/en-us/windows/uwp/app-resources/localize-strings-ui-manifest).
